tests/test_mod.py

```diff
@@ -617,7 +652,6 @@ def test_get_product_odata_full():
         for k in ret:
             assert ret[k] == expected[k]
 
-
```

Comment:
shouldn't there actually be two blank lines here? :-P

Reply:
🙈